Examples align (#75)
ArekBalysNordic/connectedhomeip: examples/all-clusters-app/nrfconnect/README.md

[nRF Connect platform overview](../../../docs/guides/nrfconnect_platform_overview.md)
to read more about the platform structure and dependencies.

> By default, the Matter accessory device has IPv6 networking disabled. You must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or Wi-Fi network. You have to make the device discoverable manually (for security reasons). See [Bluetooth LE advertising](#bluetooth-le-advertising) to learn how to do it. The controller must get the commissioning information from the Matter accessory device and provision the device into the network.

Reviewer: ```suggestion
By default, the Matter accessory device has IPv6 networking disabled. You must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or Wi-Fi network. You have to make the device discoverable manually (for security reasons). See [Bluetooth LE advertising](#bluetooth-le-advertising) to learn how to do this. The controller must get the commissioning information from the Matter accessory device and provision the device into the network.
```

To start the rendezvous, the controller must get the commissioning information
from the Matter device. The data payload is encoded within a QR code, printed to

> the UART console, and shared using an NFC tag. NFC tag emulation starts

Reviewer: ```suggestion
the UART console, and shared using an NFC tag. The emulation of the NFC tag starts
```

| [nRF5340 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF5340-DK)           | `nrf5340dk_nrf5340_cpuapp` | <details><summary>nRF5340 DK</summary><img src="../../platform/nrfconnect/doc/images/nRF5340_DK_info-medium.jpg" alt="nRF5340 DK"/></details>           |
| [nRF52840 Dongle](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-Dongle) | `nrf52840dongle_nrf52840`  | <details><summary>nRF52840 Dongle</summary><img src="../../platform/nrfconnect/doc/images/nRF52840_Dongle-medium.jpg" alt="nRF52840 Dongle"/></details> |
| [nRF7002 DK](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf7002.html#nrf7002dk-nrf5340)   | `nrf7002dk_nrf5340_cpuapp` | <details><summary>nRF7002DK</summary><img src="../../platform/nrfconnect/doc/images/nrf7002dk.jpg" alt="nRF7002 DK"/></details>    |

<hr>

### IPv6 network support

The development kits for this sample offer the following IPv6 network support for Matter:

> * Matter over Thread is supported for ``nrf52840dk_nrf52840``, ``nrf5340dk_nrf5340_cpuapp``.

Reviewer: ```suggestion
* Matter over Thread is supported for ``nrf52840dk_nrf52840`` and ``nrf5340dk_nrf5340_cpuapp``.
```

    Bluetooth LE.

-   _Short Flash Off (950ms on/50ms off)_ &mdash; The device is fully

>     provisioned, but does not yet have full Thread network, Wi-Fi network or service connectivity.

Reviewer: ```suggestion
    provisioned, but does not yet have full connectivity for Thread or Wi-Fi network, or the related services.
```
